e2e/system/build_from_git_test.go

```diff
@@ -41,51 +50,79 @@ func (s *Suite) TestBuildFromGit() {
 	s.T().Logf("Starting instance")
 	s.Require().NoError(target.Execution().Start(ctx))
 
-	s.T().Logf("Instance started")
+	s.T().Log("Instance started")
 
+	s.T().Log("Getting file bytes")
 	// The file is created by the dockerfile in the repo,
 	// so to make sure it is built correctly, we check the file
 	data, err := target.Storage().GetFileBytes(ctx, "/test.txt")
 	s.Require().NoError(err)
 
 	data = []byte(strings.TrimSpace(string(data)))
-	s.Assert().Equal([]byte("Hello, World!"), data, "File bytes do not match")
+	if !bytes.Equal([]byte("Hello, World!"), data) {
+		s.Require().NoError(err, "File bytes do not match. Expected 'Hello, World!', got '%s'", string(data))
+	}
+
+	s.T().Log("Test completed successfully")
 }
+
 func (s *Suite) TestBuildFromGitWithModifications() {
-	const namePrefix = "build-from-git-with-modifications"
+	const (
+		namePrefix = "build-from-git-with-modifications"
+		maxRetries = 3
+	)
 	s.T().Parallel()
 
 	// Setup
-	target, err := s.Knuu.NewInstance(namePrefix)
-	s.Require().NoError(err)
-
 	ctx := context.Background()
-	// This is a blocking call which builds the image from git repo
-	err = target.Build().SetGitRepo(ctx, builder.GitContext{
-		Repo:     "https://github.com/celestiaorg/knuu.git",
-		Branch:   "test/build-from-git", // This branch has a Dockerfile and is protected as to not be deleted
-		Username: "",
-		Password: "",
-	})
-	s.Require().NoError(err)
+
+	s.T().Log("Creating new instance")
+	target, err := s.Knuu.NewInstance(namePrefix)
+	if err != nil {
+		s.Require().NoError(err, "Error creating new instance")
+	}
```

Comment:
same, you don't need if statements when using require. 

```suggestion
	s.Require().NoError(err, "Error creating new instance")
```

Reply:
not resolved